libraries/AP_InertialSensor/AP_InertialSensor_Invensensev2.cpp

```diff
@@ -476,7 +480,9 @@ void AP_InertialSensor_Invensensev2::_read_fifo()
             }
             memset(rx, 0, n * INV2_SAMPLE_SIZE);
             if (!_dev->transfer(rx, n * INV2_SAMPLE_SIZE, rx, n * INV2_SAMPLE_SIZE)) {
-                hal.console->printf("INV2: error in fifo read %u bytes\n", n * INV2_SAMPLE_SIZE);
+                if (!hal.scheduler->in_expected_delay()) {
+                    hal.console->printf("INV2: error in fifo read %u bytes\n", n * INV2_SAMPLE_SIZE);
```

Comment:
should this hal.console->printf be changed to "debug("?

Reply:
changed